gradle build fails

After cloning the project, I ran `gradle build` and the following output appeared:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/michael/learning-dev/DDD-Course/first-steps-in-ddd/build.gradle' line: 13

* What went wrong:
A problem occurred evaluating root project 'first-steps-in-ddd'.
> Could not find method compile() for arguments [org.apache.commons:commons-collections4:4.4] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
```

I am not an experienced Java developer, and I have no experience with gradle, FYI.

--- a/build.gradle
+++ b/build.gradle
@@ -10,7 +10,7 @@ repositories {
 }
 
 dependencies {
-    compile 'org.apache.commons:commons-collections4:4.4'
+    implementation 'org.apache.commons:commons-collections4:4.4'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
 }

